test/extended/tests-extension/go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/modern-go/reflect2 v1.0.3-0.20250322232337-35a7c28c31ee // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/openshift/api v0.0.0-20251015095338-264e80a2b6e7 // indirect
-	github.com/openshift/library-go v0.0.0-20251015151611-6fc7a74b67c5 // indirect
+	github.com/openshift/library-go v0.0.0-20251104164011-e9c2485b059c // indirect
```

Comment:
was is updated when you did `go mod ...` in the top-level directory?

Reply:
Ah, it's probably something GoLand did behind my back 🙂

Reply:
This go.mod is actually weird. It contains unused deps and when I run `go mod`, it even removes library-go 🙃 Resolving this...